### ISSUE TITLE
the model predict a result tensor like [-0.4135, -0.3328, -0.4700, -0.4385,  0.1240]

### ISSUE THREAD

**Issue by wuchaowei2012** [tcapelle/action_recognition#11]
I find that the model does not end with the softmax layer. 

The model predicts something like [-0.4135, -0.3328, -0.4700, -0.4385,  0.1240], 

If I process tensor, something like [-0.4135, -0.3328, -0.4700, -0.4385,  0.1240], with the softmax layer, could I get the real probability of each category?

**Comment by tcapelle** [comment]
this is normal, the output of the model is a linear layer. The loss function includes the softmax layer. This is common practice.
You can just do the softmax after recovering the values.
